cfgov/unprocessed/apps/paying-for-college/css/college-costs/state-based.less

```diff
@@ -358,7 +358,7 @@ main.college-costs {
 	}
 
 	&[data-state_programprogress="0"],
-	&[data-state_programprogress="n"] {
+	&:not( [data-state_programprogress="n"] ) {
```

Comment:
Is the ":not" needed here, since a progress value of "n" equates to being a first year?

Reply:
This is probably an oversight from resolving a merge conflict! You're right, and I will push a fix!